torch/_functorch/_aot_autograd/input_output_analysis.py

```diff
@@ -460,6 +460,7 @@ def create_graph_signature(
         named_buffers=buffer_names,
         num_user_inputs=num_user_args,
         num_user_outputs=num_user_fw_outs,
+        trace_joint=trace_joint,
```

Comment:
cc: @ezyang does this look ok in terms of your refactoring work?

Reply:
The change looks reversible enough, but I can't really say I understand why you need yet another flag here. Additionally in your tests you are not even tracing out the joint!

Reply:
One thing I'll say is that for a generic "this works when torch.compile works" export frontend, these cases also need to be accounted for.